### PR TITLE
Add Bootstrap Styling to Buttons

### PR DESCRIPTION
**Pull request by thiefie**
Fixes Multiple Related Issues

#### Describe the changes you have made in this pr -
Some of the buttons did not have the bootstrap styling
### Screenshots of the changes (If any) -
nowbootstrap

### CODE REVIEW

**Comment by sakshi1499** [comment]
You have changed this for all the buttons, right? I think this changes the create assignment and post comment buttons to buttons with bootstrap. I thought the cdn was missing but you have added inline btn property right? @fluctix

**Comment by fluctix** [comment]
@sakshi1499 Yup,
Btw Bootstrap cdn was already linked in application.html , so I just styled them.

**Comment by sakshi1499** [comment]
@fluctix  I have a doubt in the html file we have added command form.submit right? So how are we actually defining the placeholder/ name of the button? I have seen this being used <%= f.submit 'A' %> for name 'A', in which file here is this name being given? How so?

**Comment by fluctix** [comment]
https://stackoverflow.com/a/3027233
https://apidock.com/rails/ActionView/Helpers/FormBuilder/submit
Check these out,
 mainly it is a method that creates a form submit button, the stackoverflow answer illustrates how it works

**Comment by sakshi1499** [comment]
I was referring to exactly these two documents. What my doubt is, is that while declaring form.submit in html file there is no mention of what the placeholder/ name is, so in which file has it been mentioned? These two links define a different style of naming the buttons namely by mentioning form.submit "PlaceholderName". @fluctix

**Comment by fluctix** [comment]
"When no value is given, it checks if the object is a new resource or not to create the proper label"(apidock)
The label is added by rails automatically if no value is specified , but in case we want a custom name using `f.submit 'ChosenName'`

**Comment by fluctix** [comment]
@PragatiVerma18 can you please review?

**Comment by fluctix** [comment]
I think the syntax is correct can you point where the problem is?

**Comment by PragatiVerma18** [comment]
`<%= f.submit 'name of button here', :class => 'submit_class_name_here' %>`
I was talking about this.

**Comment by fluctix** [comment]
I found out class: button_class does the same thing . 
Reference: https://stackoverflow.com/a/16481375
Should I change it?

**Comment by PragatiVerma18** [comment]
> I found out class: button_class does the same thing .
> Reference: https://stackoverflow.com/a/16481375
> Should I change it?

No need @fluctix. It's alright. Thanks.

**Comment by fluctix** [comment]
Anything else that needs to be done?

**Comment by PragatiVerma18** [comment]
#613 also needs similar help.

**Comment by fluctix** [comment]
@PragatiVerma18 I have fixed it along with #507

**Comment by tachyons** [comment]
Next time you create a new branch, give meaningful name

**Comment by fluctix** [comment]
i would keep that in mind next time.

File: app/views/custom_mails/_form.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </div>
 
   <div class="actions" onclick="before_submit()">
-    <%= form.submit() %>
+     <%= form.submit() nil, class: "btn btn-primary" %>
```
Comment (tachyons) [inline app/views/custom_mails/_form.html.erb]: remove `()`

File: Gemfile.lock [hunk 2]
```diff
@@ -192,8 +199,10 @@ GEM
     nio4r (2.5.2)
     nokogiri (1.10.5)
       mini_portile2 (~> 2.4.0)
+
     oauth2 (1.4.2)
       faraday (>= 0.8, < 2.0)
+
```
Comment (tachyons) [inline Gemfile.lock]: ?

File: Gemfile.lock [hunk 3]
```diff
@@ -192,8 +199,10 @@ GEM
     nio4r (2.5.2)
     nokogiri (1.10.5)
       mini_portile2 (~> 2.4.0)
+
```
Comment (tachyons) [inline Gemfile.lock]: ?

**Pull request closed without merging**